src/libinjector/linux/linux_injector.c

```diff
@@ -0,0 +1,130 @@
+#include "linux_utils.h"
+#include "linux_debug.h"
+#include "linux_shellcode.h"
+
+static event_response_t injector_int3_userspace_cb(drakvuf_t drakvuf, drakvuf_trap_info_t* info) {
+
+    injector_t injector = info->trap->data;
+
+    PRINT_DEBUG("INT3 Callback @ 0x%lx. CR3 0x%lx. vcpu %i. TID %u\n",
+                info->regs->rip, info->regs->cr3, info->vcpu, info->proc_data.tid);
+
+    if (!check_userspace_int3_trap(injector, info))
+        return VMI_EVENT_RESPONSE_NONE;
+
+    event_response_t event = VMI_EVENT_RESPONSE_NONE;
```

Comment:
no reason to set `event` value here as we are overriding it in next line with the result of `handle_shellcode`.